docs/CHANGELOG.md

```diff
@@ -14,8 +14,14 @@ The format is based on [Keep a Changelog][kc], and this project adheres to
 
 ### Added
 
+* `RenderingBundle` for full manual control of the rendering pipeline via a custom `GraphCreator` ([#1839]).
+
 ### Changed
 
+* `TextureProcessor` renamed to `TextureProcessorSystem` ([#1839)].
+* `MeshProcessor` renamed to `MeshProcessorSystem` ([#1839)].
+* `RenderingBundle` renamed to `PluggableRenderingBundle` ([#1839)].
+
```

Comment:
Please add the link to this pull request a bit under the `Fixed` title. See the previous release changelog for an example :)